src/bluefinClient.ts

```diff
@@ -589,6 +589,24 @@ export class BluefinClient {
     return signature;
   };
 
+  /**
+   * @description
+   * Gets the payload containing bytes mesasge to sign
+   * @returns SigPK
+   * */
+
+  signBytesPayloadUsingZkWallet = async (
```

Comment:
there is already a method `signPayloadUsingZkWallet`, it looks like instead of making this new method, we can just do `signPayloadUsingZkWallet = async (payload: object | Uint8Arry)`
i.e. adjusting the type to accept both and object and a uint8Array.

Reply:
actually both these function consume different methods from sui lib. Current one is to sign bytes directly. without any encoding. kept it abstract  on purpose

here is the sui library function for legacy one 

`  static async signPayloadUsingZKSignature({
        payload,
        signer,
        zkPayload
    }: {
        payload: unknown;
        signer: Signer;
        zkPayload: ZkPayload;
    }): Promise<SigPK> {
        const msgBytes = new TextEncoder().encode(JSON.stringify(payload));
        const { signature } = await signer.signPersonalMessage(msgBytes);
        const zkSignature = createZkSignature({
            userSignature: signature,
            zkPayload
        });
        return parseAndShapeSignedData({ signature: zkSignature });
    }`